### ISSUE TITLE
[Bug]: asset of types file and folder should use High Contrast mode colours

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#3376]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-asset

### Expected behavior

When running in high contrast mode the file and folder asset types should use the high contrast mode colours so they are always visible no matter the theme chosen

### Actual behavior

They use --spectrum-asset-file-background-color and do not follow high contrast mode

### What browsers are you seeing the problem in?

Chrome, Microsoft Edge

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/asset/
2. Turn on high contrast mode (either in dev tools or at Windows OS level)
3. Scroll to the file/folder images
4. Note they are a grey not one of the high contrast colors

**Comment by Westbrook** [comment]
@pfulton is Asset on the to do list for Core Tokens? Will this work be included in that update?

**Comment by pfulton** [comment]
@Westbrook I'd been told that it was going to be deprecated, but I will need to double-check that with design folks. I'll get back to you ASAP!

**Comment by jnjosh** [comment]
@Rajdeepc This seems like it's a deprecated feature, can you look into what we can do with it or if we should close?

**Comment by Rajdeepc** [comment]
Yes! I will talk with the design folks or Patrick to get an update on the decision.

**Comment by Rajdeepc** [comment]
Closed by https://github.com/adobe/spectrum-css/pull/2369
